content/applications/inventory_and_mrp/manufacturing/quality_control/quality_checks.rst

```diff
@@ -0,0 +1,77 @@
+--> Quality Checks`, and click :guilabel:`New`. On the new Quality Check form, begin by selecting an
+option from the :guilabel:`Control per` drop-down menu, either :guilabel:`Operation`,
+:guilabel:`Product`, or :guilabel:`Quantity`.
+
+Select :guilabel:`Operation` to request a check for an entire operation (ex. delivery order) and all
+products within it. Select :guilabel:`Product` to request a check for every unit of a product that
+is part of an operation (ex. every unit of a product within a delivery order).
+
+Select :guilabel:`Quantity` to request a check for every quantity of a product that is part of an
+operation (ex. one check for five units of a product within a delivery order). Selecting
+:guilabel:`Quantity` also causes a :guilabel:`Lot/Serial` drop-down field to appear, from which can
+be selected a specific lot or serial number that the quality check should be conducted for.
+
+Next, select an inventory operation from the :guilabel:`Picking` drop-down menu or a manufacturing
+order from the :guilabel:`Production Order` drop-down menu. This is necessary because Odoo needs to
+know which operation the quality check is being conducted for.
+
+If the quality check should be assigned to a specific |QCP|, select it from the :guilabel:`Control
+Point` drop-down menu. This is useful if the quality check is being created manually but should
+still be recognized as belonging to a specific |QCP|.
+
+Select a quality check type from the :guilabel:`Type` drop-down field:
+- :guilabel:`Instructions` provides specific instructions for how to conduct the quality check.
+- :guilabel:`Take a Picture` requires a picture to be attached to the check before the check can be
+completed.
+- :guilabel:`Pass - Fail` is used when the product being checked must meet a certain criteria to
+pass the check.
+- Selecting :guilabel:`Measure` causes a :guilabel:`Measure` input field to appear, in which a
+measurement must be entered before the check can be completed.
+- Selecting :guilabel:`Worksheet` causes a :guilabel:`Quality Template` drop-down field to appear,
+from which can be selected a quality worksheet that must be filled out to complete the check.
+
+In the :guilabel:`Team` field, select the quality team that is responsible for the quality check. In
+the :guilabel:`Company` field, select the company that owns the product being inspected.
+
+On the :guilabel:`Notes` tab at the bottom of the form, enter any relevant instructions in the
+:guilabel:`Instructions` text entry box (ex. attach a picture of the product). Add any other
+information that needs to be included in the :guilabel:`Notes` text entry field.
+
+Process quality check
+=====================
+
+Quality checks are most commonly processed by selecting the :guilabel:`Quality Checks` button on an
+inventory operation or manufacturing order, or within the tablet view for a specific work order.
+However, it is also possible to process a quality check directly on the check itself.
```

Comment:
This information is sufficient, especially since we currently have no documentation on quality checks. For the future, I feel like this section can be expanded upon with images of examples of when the quality checks smart button becomes available.

Reply:
```suggestion
Quality checks are most commonly processed by selecting the :guilabel:`Quality Checks` smart button
on an inventory operation or manufacturing order, or within the tablet view for a specific work
order. However, it is also possible to process a quality check directly on the check itself.
```
Adding "smart" button